package.json

```diff
@@ -309,6 +309,23 @@
             "markdownDescription": "**Experimental**: Run `swift build` in the background whenever a file is saved. It is possible the background compilation will already be running when you attempt a compile yourself, so this is disabled by default.",
             "order": 10
           },
+          "swift.backgroundIndexing": {
+            "type": "boolean",
+            "default": false,
+            "markdownDescription": "Enable or disable background indexing. This option will not work for Swift versions prior to 6.0.",
+            "order": 11
+          },
+          "swift.backgroundPreparationMode": {
+            "type": "string",
+            "enum": [
+              "build",
+              "noLazy",
+              "enabled"
+            ],
+            "default": "enabled",
+            "markdownDescription": "Determines how background indexing should prepare a target. This option will not work for Swift versions prior to 6.0. Possible values are:\n- `build`: Build a target to prepare it\n- `noLazy`: Prepare a target without generating object files but do not do lazy type checking and function body skipping\n- `enabled`: Prepare a target without generating object files and the like",
```

Comment:
Using "enumDescriptions" is the typical vscode pattern instead of \n descriptions